internal/dkg/store.go

```diff
@@ -17,8 +17,8 @@ import (
 	"github.com/drand/drand/v2/common/log"
 )
 
-type boltStore struct {
-	sync.RWMutex
+type BoltStore struct {
+	lock          sync.RWMutex
```

Comment:
why the need for explicit field for it?

Reply:
linter complained about embedding Mutex in public structs

Reply:
Ah right, does this one really need to be public? Although it's in our internal folder, so it's fine. We can had a rule to our linter to accept these in `internal` I think.

Reply:
`unexported-return: exported func NewDKGStore returns unexported type *dkg.boltStore, which can be annoying to us` I think because I use it in the dkg cli